src/main/java/com/dougnoel/sentinel/steps/VerificationSteps.java

```diff
@@ -144,30 +306,41 @@ public static void verifyElementHasClass(String elementName, String assertion, S
     public static void verifyElementAttributeHasValue(String elementName, String attribute, String assertion, String matchType, String value) {
```

Comment:
This is an internal helper method with no attached cucumber step. Also, get rid of the match type and the associated logic. I don't think we need it.

```suggestion
    private static void verifyElementAttributeHasValue(String elementName, String attribute, String assertion, String value) {
```

Reply:
I'm confused. The only class with this is indeed attached to a step. I'm fairly sure I didn't write it afterwards either.